### PR TITLE
Added coverage

### PR DESCRIPTION
**Pull request by nwu63**
## Purpose
Added codecov coverage, and also enforced that all tests on the private image should pass.

## Type of change
What types of change is it?
_Select the appropriate type(s) that describe this PR_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (non-backwards-compatible fix or feature)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Documentation update
- [x] Maintenance update
- [ ] Other (please describe)

### CODE REVIEW

File: .github/build_real.sh [hunk 1]
```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
 set -e
-pip install .[testing]
+pip install -e .[testing]
```
Comment (marcomangano) [inline .github/build_real.sh:3]: I guess my question for [this PR](https://github.com/mdolab/multipoint/pull/16) comes from here: why is this necessary / best practice?

Comment (marcomangano) [reply .github/build_real.sh:3]: I have been redirected to the instructions, and I see how this makes sense... but at the same time it looks like it is not necessary for the IDWarp PR, I am getting a bit lost

Comment (nwu63) [reply .github/build_real.sh:3]: Interesting, perhaps this is not needed..

**Review by marcomangano — APPROVED** [review]
